src/ow1/heroes/echo.opy

```diff
@@ -15,3 +29,19 @@ def initEcho():
     @Name "[echo.opy]: initEcho()"
     # [TODO]
     pass
+
+    
+rule "[echo.opy]: Set Life for echo when duplicating a tank":
+    @Event eachPlayer
+    @Hero echo
+    @Condition eventPlayer.isDuplicatingAHero()
+    @Condition eventPlayer.getHeroOfDuplication() in getTankHeroes()
+    
+    eventPlayer.hero_id_duplicated = heroID(eventPlayer.getHeroOfDuplication())
+    clearCustomHealth()
+    setCustomHealth(
+        health_pool[eventPlayer.hero_id_duplicated][0],
+        health_pool[eventPlayer.hero_id_duplicated][1],
+        health_pool[eventPlayer.hero_id_duplicated][2],
+    )
+    waitUntil(eventPlayer.isDuplicatingAHero() == false, 9999)
```

Comment:
Why do you have to wait here?

Reply:
It was for testing purpose, I forgot to remove it.